types/rower.model.ts

```diff
@@ -0,0 +1,25 @@
+export interface Rower {
+	id: string;
+	regatta_id: string;
+	club_id: string;
+	knrb: string;
+	knrb_num: null; // Is dit hetzelfde als 'knrb'?
```

Comment:
Nee andere nummers

Reply:
knrb_num is het oude systeem en werkt alleen voor Nederlanders

Reply:
Maar knrb_num is nog wel nodig dus?

Reply:
Het is wel nodig aangezien een wedstrijdleiding wel een knrb_num mag invullen maar geen knrb.